content/pages/home/header.json

```diff
@@ -1,16 +1,14 @@
 {
   "body": "Find or publish datasets and AI algorithms for solving challenges in mobility. Experience and leverage decentralized technologies and the European Gaia-X data sharing concepts. Facilitating public-private, cross-sector, cross-border collaboration while maintaining digital sovereignty.",
   "cta": [
+    { "label": "Learn more", "link": "#" },
     {
-      "label": "Go to catalogue",
+      "label": "View catalogue",
       "link": "/search?sort=_score&sortOrder=desc&text="
-    },
-    { "label": "About this demo", "link": "/about" }
+    }
   ],
   "carousel": [
-    "/images/headerCarousel/1-road.jpeg",
-    "/images/headerCarousel/2-bikes.jpeg",
-    "/images/headerCarousel/3-ev-charge.jpeg",
-    "/images/headerCarousel/4-e-scooter.jpeg"
+    "/images/headerCarousel/1-Energy-SHR.png",
```

Comment:
can find the place in the code, but the carousel looks very washed. if you inspect it and disable

--image-height: 50rem, the colors looks exactly like the design. not sure what the cause is

Reply:
In the last meeting it was decided to have the carousel working but repeating the same picture